pages/documentation/developer/api_guide/limits.md

```diff
@@ -17,4 +20,28 @@ CurrentLimits currentLimits = network.getDanglingLine("DL").newCurrentLimits()
     .endTemporaryLimit()
     .add();
 ```
-A `NaN` value for the acceptable duration of a temporary limit means an infinite duration.
```

Comment:
I removed that line deliberately, as we should not encourage the use of an infinite duration. A temporary limit with an infinite duration is nothing more than a permanent limit, and if such a temporary limit exists then problems/conflicts begin